component/Card/index.html

```diff
@@ -0,0 +1,264 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>css assignment</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <!-- text only card -->
+    <div class="card">
+      <div class="card-background">
+        <div class="card-body">
+          <h3 class="card-title">Title</h3>
+          <small class="small-text">secondary text</small>
+          <p class="card-description">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Voluptatem quis, expedita ab saepe voluptatum ut ad sequi, quasi dolores enim animi fugit nostrum autem quaerat libero doloremque! Impedit, asperiores dolores.</p>
+        </div>
```

Comment:
You can use prettier to make your indentation proper

Reply:
done